### PR TITLE
Visual monitor tests for homepage & auction page

### PR DESCRIPTION
**Pull request by miri**

### CODE REVIEW

File: .idea/scopes/scope_settings.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<component name="DependencyValidationManager">
```
Comment (amitaibu) [inline .idea/scopes/scope_settings.xml]: remove `.idea` and add it to `.gitignore`

File: visual-monitor/test/tests.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+  it('should show the home page',function(done) {
+    client
+      .url(baseUrl)
+      .pause(9000)
+      .webdrivercss(testName + '.homepage', {
+        name: '1',
+        exclude:
+            [
+                // Top carousel.
```
Comment (amitaibu) [inline visual-monitor/test/tests.js]: 2 spaces, not tabs.

File: visual-monitor/test/tests.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            .url(baseUrl + 'node/19640')
+            .webdrivercss(testName + '.auction46', {
+                name: '1',
+                exclude:
+                    [
+                        // Sale count down.
+                        '.pane-circuit-sale-circuitcountdown',
+                    ],
+                remove: [],
+                screenWidth: selectedCaps == 'chrome' ? [640, 960, 1200] : undefined,
```
Comment (amitaibu) [inline visual-monitor/test/tests.js]: remove the `screenWidth` property for now on all tests

**Comment by amitaibu** [comment]
some images are wrong

**Comment by mldk-tech** [comment]
@amitaibu - Can I merge?

**Comment by amitaibu** [comment]
Yes, please merge squash

**Comment by mldk-tech** [comment]
Done.

**Pull request closed without merging**